src/python/grpcio/grpc/_channel.py

```diff
@@ -85,7 +85,7 @@ def _deadline(timeout):
 
 
 def _unknown_code_details(unknown_cygrpc_code, details):
-  return b'Server sent unknown code {} and details "{}"'.format(
+  return 'Server sent unknown code {} and details "{}"'.format(
```

Comment:
I guess the question also applies to all such similar changes in the remainder of this file.